static_src/util/cf_api.js

```diff
@@ -15,6 +16,25 @@ import userActions from '../actions/user_actions.js';
 
 const APIV = '/v2';
 
+function handleError(err, errHandler) {
+  let handler = errHandler;
```

Comment:
you should initialize this to `errorActions.errorFetch` and the following conditional should check for the existence (instead of the absence) of `errHandler` and assign `errHandler` to `handler` if it exists